client/state/ui/guided-tours/contexts.js

```diff
@@ -34,6 +34,15 @@ export const isEnabled = feature => () =>
 	config.isEnabled( feature );
 
 /**
+ * Returns true if there is a global `isDebuggingGuidedTours` present with a
+ * truthy value.
+ *
+ * @return {Boolean} True if `window.isDebuggingGuidedTours` truthy
+ */
+export const isDebuggingGuidedTours = () =>
```

Comment:
Yeah, that sounds great. I was intentionally avoiding more involved
solutions such as anything using `localStorage`, etc., but cookies
sound like just the right tool. I'll give that a go, thanks!